test/unit/fixtures/fake_project_with_sonar_properties_file/sonar-project.properties

```diff
@@ -1,4 +1,5 @@
 sonar.projectKey=foo
 sonar.projectVersion=1.0-SNAPSHOT
 sonar.projectName=Foo
-sonar.sources=.
\ No newline at end of file
+#sonar.token=ignore-this
```

Comment:
is this needed?

Reply:
it's needed to fully cover [this line](https://github.com/SonarSource/sonar-scanner-npm/pull/111/files#diff-c0e4912587622579192cb35813490cac16223f5edb98b8e72d7aed68fc8ffc11R168)